### PR TITLE
frontend/feat #2252 dynamic country code added in YS

### PR DESCRIPTION
**Pull request by S-PRAVINKUMAR**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/android-native/app/src/main/res/drawable/ny_ic_whatsapp_logo.png
Comment (kavya-shree-s) [inline Frontend/android-native/app/src/main/res/drawable/ny_ic_whatsapp_logo.png:1]: @S-PRAVINKUMAR Please reduce the size if possible

File: Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs [hunk 1]
```diff
@@ -119,33 +124,33 @@ eval (MobileNumberButtonAction PrimaryButtonController.OnClick) state = do
...
             pure $ hideKeyboardOnNavigation true
             else pure unit
-    let isValidMobileNumber = case (charAt 0 value) of
-                                    Just a -> if a=='0' || a=='1' || a=='2' || a=='3' || a=='4' then false
-                                                else if a=='5' then
-                                                    if value=="5000500050" then true else false 
-                                                        else true 
-                                    Nothing -> true 
-    if (length value == 10 && isValidMobileNumber) then do 
-        let _ = unsafePerformEffect $ logEvent state.data.logField "ny_user_mobnum_entry"
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs:140]: @S-PRAVINKUMAR Don't remove any logevent

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

**Comment by kavya-shree-s** [comment]
Squash the commit and Add screen recording

**Comment by S-PRAVINKUMAR** [comment]
> Squash the commit and Add screen recording

[Screen Record](https://drive.google.com/file/d/12VcoASz7qv2GBOquJFK3ZYwAV6oZPx6W/view?usp=share_link)

in record It seems like slow. but in real device it is not like that

File: Frontend/ui-customer/package.json [hunk 2]
```diff
@@ -48,12 +48,12 @@
...
     "node-notifier": "^10.0.1",
     "npm-run-all": "^4.1.5",
+    "purty": "^7.0.0",
     "semantic-release": "^19.0.5",
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10",
     "webpack-dev-server": "^3.10.1",
-    "webpack-merge": "^5.8.0",
-    "purty": "^7.0.0"
+    "webpack-merge": "^5.8.0"
```
Comment (ashishsingh101) [inline Frontend/ui-customer/package.json:56]: why we need this ?

**Review by ashishsingh101 — APPROVED** [review]

File: Frontend/ui-customer/package.json (hunk 2 above)
Comment (S-PRAVINKUMAR) [reply Frontend/ui-customer/package.json]: it is already present only. just reordered. changed now

**Review by kavya-shree-s — APPROVED** [review]

**Pull request merged**